curator/api/utils.py

```diff
@@ -88,11 +89,18 @@ def index_closed(client, index_name):
     :arg index_name: The index name
     :rtype: bool
     """
-    index_metadata = client.cluster.state(
-        index=index_name,
-        metric='metadata',
-    )
-    return index_metadata['metadata']['indices'][index_name]['state'] == 'close'
+    if get_version(client) >= (1, 5, 0):
+        indices = client.cat.indices(index=index_name, format='json', h='status')
+        if not isinstance(indices, list):  # content-type: text/plain
```

Comment:
Please add a comment here to explain that this workaround is for AWS Elasticsearch, that it is using `text/plain` to respond.